hw-3__danya.js

```diff
@@ -0,0 +1,122 @@
+const tr='true'
+fal=false
+sev=17
+und=undefined
+n=null;
+console.log(typeof tr, typeof fal, typeof sev, typeof und, typeof null)
+
+
+let height = 15
+let width = 20
+if (height>width){
+    console.log(height)
+}else if(width>height){
+    console.log(width)
+}
+
+
+for(let i=1; i<=20; i++){
+    if(i%3===0){
+        console.log(i)
+    }
+}
+
+
+
+let key = true
+    let documents = true
+    let pen = true
+    let apple = false
+    let orange = true
+    let shouldGoToWork
+    if(documents&&pen&&key&&(apple||orange)){
```

Comment:
here you can use just "shouldGoToWork =documents&&pen&&key&&(apple||orange); " without condition (if else)